<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:/* qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #cda8a1, stop: 1 #cdf833); /* Chrome10+,Safari5.1+ */
rgb(181, 233, 255)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_14">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>ABR</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_7">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,150, 220, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>Insert</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(222,120, 0, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>33</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,150, 220, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>Supp</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(222,120, 0, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>33</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,150, 220, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>Chercher</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_3">
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(222,120, 0, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>33</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>AVL</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_8">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QLineEdit" name="lineEdit_13">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>222</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Serif</family>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 170, 127)</string>
         </property>
         <property name="text">
          <string>0 ms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,150, 220, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>Insert</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(222,120, 0, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>22</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,150, 220, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>Supp</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_5">
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(222,120, 0, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>22</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,150, 220, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>Chercher</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_6">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(222,120, 0, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>22</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>BICOLOR</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_12">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QLineEdit" name="lineEdit_14">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Serif</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 170, 127)</string>
         </property>
         <property name="text">
          <string>0 ms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,150, 220, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>Insert</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_9">
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(222,120, 0, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>44</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,150, 220, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>Supp</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_10">
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(222,120, 0, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>44</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_9">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,150, 220, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>Chercher</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_11">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(222,120, 0, 255), stop:1 rgba(255, 255, 255, 255))</string>
             </property>
             <property name="text">
              <string>44</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_10">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>111</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>M</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
